packaging/CMakeLists.txt

```diff
@@ -178,6 +178,7 @@ elseif(APPLE)
 	set_target_properties(Mapper PROPERTIES
 	  MACOSX_BUNDLE_INFO_STRING          "${CPACK_PACKAGE_DESCRIPTION_SUMMARY}"
 	  MACOSX_BUNDLE_ICON_FILE            "Mapper.icns"
+    MACOSX_BUNDLE_INFO_PLIST           "${PROJECT_SOURCE_DIR}/packaging/apple/Info.plist"
```

Comment:
Thanks, Francesco, for the code contribution! I'll leave this pull request review to Kai as I know absolutely nothing about Mac. However, the whitespace error is glaring, and cannot help myself not to point it out. ;-)

The line should start with a tab and two spaces for proper alignment. Below is an illustration how a properly formatted line would look like in Qt Creator with whitespace visualization on.
![image](https://user-images.githubusercontent.com/5584406/112476268-02575c80-8d72-11eb-8ca8-994de8ec445f.png)